content/news/2023-10-21-bevy-0.12/index.md

```diff
@@ -0,0 +1,1638 @@
+* `Time<()>`
+
+`Time<Real>` measures the true, unedited frame and app durations. For diagnostics/profiling, use that one. It's also used to derive the others. `Time<Virtual>` can be sped up, slowed down, and paused, and `Time<Fixed>` chases `Time<Virtual>` in fixed increments. Lastly, `Time<()>` is automatically overwritten with the current value of `Time<Fixed>` or `Time<Virtual>` upon entering or exiting `FixedUpdate`. When a system borrows `Time`, it actually borrows `Time<()>`.
+
+Try the new `time` example to get a better feel for these resources.
+
+The fix for the windup problem was limiting how much `Time<Virtual>` can advance from a single frame. This then limits how many times `FixedUpdate` can be queued for the next frame, and so things like frame lag or your computer waking up from a long sleep can no longer cause a death spiral. So now, the app won't freeze, but things happening in `FixedUpdate` will appear to slow down since it'll be running at a temporarily reduced rate.
+
+## Reduced Tracing Overhead
+
+<div class="release-feature-authors">authors: @hymm, @james7132</div>
+
+Bevy uses the [tracing](https://crates.io/crates/tracing) library to measure system running time (among other things). This is useful for determining where bottlenecks in frame time are and measuring performance improvements. These traces can be visualized using the [tracy](https://github.com/wolfpld/tracy) tool. However, using tracing's spans has a significant overhead to it. A large part of the per-span overhead is due to allocating the string description of the span. By caching the spans for systems, commands, and parallel iteration, we have significantly reduced the CPU time overhead when using tracing. In the PR that introduced system span caching, our "many foxes" stress test went from 5.35 ms to 4.54 ms. In the PR that added caching for the parallel iteration spans, our "many cubes" stress test went from 8.89 ms to 6.8 ms.
+
+![tracing overhead](tracing-overhead-reduction.png)
+
+## Pitch Audio Source
+
+<div class="release-feature-authors">authors: @basilefff</div>
+
+Audio can now be played by pitch, which is useful to debug audio issues, to use as a placeholder, or for programmatic audio.
+
+A `Pitch` audio source can be created from its frequency and its duration, and then be used as a source in a `PitchBundle`.
+
+```rust
+fn play_pitch(
+    mut pitch_assets: ResMut<Assets<Pitch>>,
+    mut commands: Commands,
+) {
+    // This is a A for 1 second
+    let pitch_handle = pitch_assets.add(Pitch::new(440.0, Duration::new(1, 0)));
+    // Play it now
+    commands.spawn(PitchBundle {
+        source: pitch_handle,
+        ..default()
+    });
+}
+```
+
+Audio is generated at the given frequency using a [sine wave](https://en.wikipedia.org/wiki/Sine_wave#Audio_example). More complex sounds can be created by playing several pitch audio sources at the same time, like chords or hamonics.
+
+## Automatic Batching/Instancing of Draw Commands
```

Comment:
This is what we need to link to.

Reply:
Resolving as we already link to this. We can use the comment to resolve the markdownlint behavior mismatch.